dependencies.gradle

```diff
@@ -1,9 +1,12 @@
 dependencies {
     compile("com.github.GTNewHorizons:GT5-Unofficial:5.09.40.17:dev")
+    compile("com.github.GTNewHorizons:StructureLib:1.0.14:dev")
     compile("com.github.GTNewHorizons:TinkersConstruct:1.9.0.10-GTNH:dev")
+    compile("com.github.GTNewHorizons:Mantle:0.3.4:dev")
```

Comment:
Why do yo specify StructureLib and Mantle here? They are not referenced in GC code and `compile` pulls them in anyway.

Reply:
Also, this will conflict with https://github.com/GTNewHorizons/Galacticraft/pull/44, I'd advise not touching this file for the PR.

Reply:
oh, I didn't see second comment